pkg/batch/subscriber/processor/metrics.go

```diff
@@ -68,11 +69,21 @@ var (
 			Name:      "subscriber_handled_event_total",
 			Help:      "Total number of handled messages",
 		}, []string{"subscriber", "code"})
+
+	pushHandledHistogram = prometheus.NewHistogramVec(
```

Comment:
How about creating a common histogram for the subscribers, like the `subscriberHandledCounter` and the `subscriberReceivedCounter` for future subscribers that use histogram?

Reply:
Sure! 
It make sense, maybe there will be some other subscribers need to report this metric, for now, only push sender.

Reply:
I have changed it's name to `subscriberHandledHistogram`, ref: https://github.com/bucketeer-io/bucketeer/pull/995/files#diff-d04432152ef4b131259a7c41a96b4bbe74d34e8e8d24dfed546a82dd367c86feR71-R74